changelogs/changelog.yaml

```diff
@@ -197,3 +197,31 @@ releases:
     - 211-fix-error-handling-during-tagging-failure.yaml
     - 53-ec2_vol-boto3-port.yml
     release_date: '2020-12-10'
+  1.4.0:
+    changes:
+      bugfixes:
+      - ec2_vol - a creation or update now returns a structure with an up to date
+        list of tags (https://github.com/ansible-collections/amazon.aws/pull/241).
+      minor_changes:
+      - aws_ec2 - Add hostname options concatenation
+      - aws_ec2 inventory plugin - avoid a superfluous import of ``ansible.utils.display.Display``
+        (https://github.com/ansible-collections/amazon.aws/pull/226).
+      - aws_ec2 module - Replace inverse aws instance-state-name filters !terminated,
+        !shutting-down in favor of postive filters pending, running, stopping, stopped.
+        Issue 235. (https://github.com/ansible-collections/amazon.aws/pull/237)
```

Comment:
Just a comment for future - I missed this and I guess @goneri @gravesm and @jillr missed it too, but this should have been a bugfix rather than a minor_change

Reply:
@tremble  I also should have backed out the changelog entry about the boto3, I'll PR a changelog update real quick here